persistence/sql/persister_grant_jwk.go

```diff
@@ -90,7 +90,7 @@ func (p *Persister) GetPublicKey(ctx context.Context, issuer string, subject str
 	var data trust.SQLData
 	query := p.Connection(ctx).
 		Where("issuer = ?", issuer).
-		Where("subject = ?", subject).
+		Where("subject = ? or subject = ''", subject).
```

Comment:
I would prefer a separate SQL key `allow_any_subject` and then: `WHERE subject = ? OR allow_any_subject IS TRUE`

Reply:
Okay, it's also simpler and easier to understand!